src/hooks.ts

```diff
@@ -61,7 +61,7 @@ export function useResponsiveSVGSelection<T>(
         .selectAll('*')
         .remove();
     };
-  }, [initialSize, minSize]);
+  }, [initialSize[0], initialSize[1], minSize[0], minSize[1]]);
```

Comment:
Thanks for the fix! Ideally consumers of the component can manage and ensure the array references don't change, but in reality the component solving this would be a convenience for people who are not able to ensure that.

Not sure if `[...initialSize, ...minSize]` is a better syntax here.  I think I prefer the explicitness here.

Feel free to update the `package.json` (patch version) and update the `CHANGELOG.md`, otherwise I can do that later.

Reply:
This might cause an issue, as far as I know the dependency array length needs to stay the same. So if someone had no minSize and then passed it, would change from 0 to 2 and might throw a warning